### ISSUE TITLE
ANDROID_HOME not picked up on remote machine

### ISSUE THREAD

**Issue by PaulWoitaschek** [gojuno/mainframer#161]
Even though `ANDROID_HOME` is set up on the remote machine

```
echo $ANDROID_HOME
/opt/android-sdk
```

The build fails with a
```
* What went wrong:
A problem occurred configuring project ':audiobook'.
> SDK location not found. Define location with sdk.dir in the local.properties file or with an ANDROID_HOME environment variable.
```

When I now ssh into the remote machine and set the local.properties with
`sdk.dir=/opt/android-sdk`
it builds fine.

**Comment by artem-zinnatullin** [comment]
Please carefully read this section https://github.com/gojuno/mainframer/blob/development/docs/SETUP_REMOTE.md#environment

Hope that'll help! Feel free to comment/reopen.

**Comment by PaulWoitaschek** [comment]
I can't reopen. 
My bashrc now looks like

```
[ph1b@ph1b ~]$ cat .bashrc 
#
# ~/.bashrc
#

export ANDROID_HOME="/opt/android-sdk"

# If not running interactively, don't do anything
[[ $- != *i* ]] && return
```
But it's still the same. Do I overread something?

**Comment by artem-zinnatullin** [comment]
Are you sure you're not reusing old ssh connection?

**Comment by PaulWoitaschek** [comment]
Yes I'm sure. Rebootet several times since then.

**Comment by artem-zinnatullin** [comment]
What OS is on your remote machine? We use Ubuntu Server 16.04.2 LTS on remote machine and solution from [SETUP_REMOTE](https://github.com/gojuno/mainframer/blob/development/docs/SETUP_REMOTE.md#environment).

I also have machine with `zsh` as default shell and `~/.zshenv` works for both interactive and non-interactive shells on it.

**Comment by PaulWoitaschek** [comment]
It's my Arch Linux desktop.

**Comment by artem-zinnatullin** [comment]
Then I'm not sure since I'll be my assumptions on googling, haven't use
Arch, if you'll find something — please post it here!

**Comment by ghostbuster91** [comment]
I think that this issue and #177 are the same. And here is the reason: https://stackoverflow.com/questions/216202/why-does-an-ssh-remote-command-get-fewer-environment-variables-then-when-run-man

TLDR:
Neither `~/.bash_profile` nor  `~/.bash_login` nor  `~/.profile` are read when using bash in non-interactive mode.
